### PR TITLE
docs(README): add about operation checks and guarantees

### PR DESCRIPTION
**Pull request by hal-shu-sato**
## Description

- Add information about operation checks and guarantees.

## Motivation and Context

We test only on Windows. I would like to state it clearly.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: README.en.md [hunk 1]
```diff
@@ -53,6 +53,8 @@ After launching the AviUtl Package Manager, please perform the following setting
 ### Other
 
 - If a package cannot be installed using the above method, it can be installed using the traditional method of manually copying the files
+- We have tested the Windows version only; we do not guarantee the operation of the Mac or Linux versions
+  - But if you encounter any problems, please create an Issue and report it to us
```
Comment (yumetodo) [inline README.en.md:57]: ButはMac or Linux versionに対して修飾したいと思われますが、そうは読めないように思います。

File: README.en.md [hunk 2]
```diff
@@ -53,6 +53,8 @@ After launching the AviUtl Package Manager, please perform the following setting
 ### Other
 
 - If a package cannot be installed using the above method, it can be installed using the traditional method of manually copying the files
+- We have tested the Windows version only; we do not guarantee the operation of the Mac or Linux versions
```
Comment (yumetodo) [inline README.en.md:56]: >we do not guarantee the operation of 

We are not supporting

**Comment by hal-shu-sato** [comment]
I fixed it.

**Review by yumetodo — APPROVED** [review]

**Review by mitosagi — COMMENTED** [review]
There is no need for me to review it.

**Pull request merged**